pset-functions.js

```diff
@@ -114,6 +181,41 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+// creating palindrome function
+function isPalindrome(word) {
+
+  // creating result variable
+  let result
+
+  // making sure we're dealing with a string
+  if (typeof word === "string") {
+    word = word.toLowerCase()
+  } else {
+    return "isPalindrome() requires a string parameter."
+  }
+
+  // palindrome logic
+  for (i = 0; i < (word.length / 2); i++) {
+    if (word[i] === word[word.length - (1 + i)]) {
```

Comment:
Nate, brilliant solution! You are checking both ends and meeting in the middle (without using built in methods)! If you did want to make use of some of javascript's built in methods, please find an alternate solution [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L163)